components/server/ee/src/workspace/workspace-factory.ts

```diff
@@ -207,6 +218,7 @@ export class WorkspaceFactoryEE extends WorkspaceFactory {
     protected async createForPrebuiltWorkspace(
         ctx: TraceContext,
         user: User,
+        organization: Organization | undefined,
```

Comment:
We don't need the full Organization here, we only need the ID.

What's the reason to keep it as `undefined`? I'd have expected that we want all workspaces to belong to exactly one Org.

Reply:
> What's the reason to keep it as undefined? I'd have expected that we want all workspaces to belong to exactly one Org.
We still have users running personal projects. I'm checking the case [here](https://github.com/gitpod-io/gitpod/pull/16261/files/83724fab37620f1a0edf2003fdfd8aabfa5cdf0b#diff-5eede9fcdcf767a1a542eb64c8eba6415160606ec91fe5563184237af87dceebR49)